File: sdks/thunkmetrc-java/wrapper/src/main/java/io/github/thunkier/thunkmetrc/wrapper/models/CreateDeliveriesRetailerEndRequestPackagesItem.java
package io.github.thunkier.thunkmetrc.wrapper.models;
import com.fasterxml.jackson.annotation.JsonProperty;public class CreateDeliveriesRetailerEndRequestPackagesItem {
    @JsonProperty("EndQuantity")
    public Integer endQuantity;
    @JsonProperty("EndUnitOfMeasure")
    public String endUnitOfMeasure;
    @JsonProperty("Label")
    public String label;
}

File: sdks/thunkmetrc-kotlin/wrapper/src/main/kotlin/io/github/thunkier/thunkmetrc/wrapper/models/CreateDeliveriesRetailerEndRequestPackagesItem.kt
package io.github.thunkier.thunkmetrc.wrapper.models

import com.fasterxml.jackson.annotation.JsonProperty



data class CreateDeliveriesRetailerEndRequestPackagesItem(
    @JsonProperty("EndQuantity")
    val endQuantity: Int? = null,
    @JsonProperty("EndUnitOfMeasure")
    val endUnitOfMeasure: String? = null,
    @JsonProperty("Label")
    val label: String? = null
)
